packages/docusaurus-utils-validation/src/validationSchemas.ts

```diff
@@ -96,6 +96,9 @@ export const PathnameSchema = Joi.string()
     '{{#label}} is not a valid pathname. Pathname should start with slash and not contain any domain or query string.',
   );
 
+// joi empty string not allowed by default. so using empty('') for custom use
+export const RouteBasePathSchema = Joi.string().empty('').default('');
```

Comment:
Let's wait for a decision to be made in https://github.com/facebook/docusaurus/issues/8254#issuecomment-1294681922

I'd prefer to have `allow('')` instead of `empty('')` here, and not have a default value of '' => default should applied on a case by case basis.

Eventually we could use this scema for baseUrl too (rename to PathSegmentSchema?), and normalize both baseUrl + routeBasePath to have leading/trailing slashes

Reply:
When i did some of test, if use `allow('')`, it return '' 

So i use empty('') then default('')

It gauarantee this case

`joi.string().allow('').default('/') => ''`
`joi.string().empty('').default('').default('/') => '/'`

`joi.string().empty('').default('/') => '/'` => this case can't gaurantee because it return ''

I found rootBasePath used '/', '', 'blogs' in this repo and then customized by default value individually

Reply:
You forgot I initially suggested to add the same code as baseurl: `.custom((value: string) => addLeadingSlash(addTrailingSlash(value))),`

Reply:
i also test this case you announced,
 
when i input empty string ""
`joi.string().custom((value: string) => addLeadingSlash(addTrailingSlash(value))) => throw error ` 
`joi.string().allow('').custom((value: string) => addLeadingSlash(addTrailingSlash(value)) => return ''

Reply:
example
![스크린샷 2022-10-28 오후 7 35 03](https://user-images.githubusercontent.com/49237205/198568351-891f7b41-48c5-44be-925d-3244bd2f20ad.png)
![스크린샷 2022-10-28 오후 7 38 10](https://user-images.githubusercontent.com/49237205/198568356-c511c678-3f13-4a68-869b-ea4d02a5c00a.png)